src/components/game/GameBreakPage.js

```diff
@@ -6,11 +6,13 @@ const GameBreakPage = ({ onContinue }) => {
   return (
     <div className="break-content">
       <h2>Hienoa!</h2>
-      <p>
+      <p className="break-page-paragraph-1">
```

Comment:
Tykkäsin enemmän vanhasta tavasta verrattuna tähän eksplisiittiseen classNameen, mutta CSS is so fucked että tää ehk on ihan hyvä ratkasu meidän tapaukses :D

Reply:
No siis tää on just se millä noi saa korjattua :D ei tää hieno oo mut it fixes things